Training/Program.cs

```diff
@@ -1,7 +1,43 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+string inputStr, inSplChar;
+for (; ; ) {
+   Console.Write ("Enter a string: ");
+   inputStr = Console.ReadLine ();
+   if (inputStr == "")
+      Console.WriteLine ("Enter a valid string. Empty string is not allowed");
+   else
+      break;
 }
+for (; ; ) {
+   Console.Write ("Mention the special character to be swapped: ");
+   inSplChar = Console.ReadLine ();
+   if (inSplChar == "")
+      Console.WriteLine ("Enter a valid special character. Entering empty special character is not allowed");
+   else
+      break;
+}
+Console.Write ("Mention the order: ");
+string order = Console.ReadLine ();
+SortAndSwapSplChars (inputStr, inSplChar, order);
+
+static void SortAndSwapSplChars (string inputStr, string inSplChar, string order = "") {

```

Comment:
Optional param 'order' is not meaningful as it is used only one time and you've passed the param too. Figure it out whether there is another way around.